### PR TITLE
[#145024711 ] Reduce http_tester instance memory

### PR DESCRIPTION
**Pull request by combor**
## What

As we run tests in parallel we deploy multiple instances of http_tester
app at once. This may have an impact on the platform so we decided to
reduce available instance memory to 64m

## How to review

Run the pipeline and check if tests pass. 

## Who can review
Not @combor

### CODE REVIEW

**Comment by dcarley** [comment]
Because we're attempting to fix a specific problem, could you tag this PR with the story ID [#145024711](https://www.pivotaltracker.com/story/show/145024711) and include the error message that we've been seeing in the commit message?

**Comment by combor** [comment]
Done. BTW note this is still WIP.

**Comment by dcarley** [comment]
Just getting my requests in early so that it doesn't hold up the review 😀

File: platform-tests/src/acceptance/http_edge_cases_test.go [hunk 1]
```diff
@@ -65,10 +62,8 @@ var _ = Describe("HTTP edge cases", func() {
 			appName = generator.PrefixedRandomName("CATS-APP-HTTP-TESTER-")
 			Expect(cf.Cf(
 				"push", appName,
-				"-b", config.GoBuildpackName,
 				"-p", "../../example-apps/http_tester",
 				"-d", config.AppsDomain,
-				"-c", "./bin/http_tester; sleep 1; echo 'done'",
```
Comment (alext) [inline platform-tests/src/acceptance/http_edge_cases_test.go]: I don't think this is picking up the `manifest.yml`. I think the cf CLI always looks in the current directory for that (even when `-p` is specified), so you'll need to add a `"-f", "../../example-apps/http_tester/manifest.yml",` line.
